### ISSUE TITLE
Control Mirror Folding

### ISSUE THREAD

**Issue by AlanJohnWilliams** [bretterer/home-assistant-rivian#137]
Would love the ability to control mirror folding. E.g. when at home and the truck has been parked for X minutes, automatically fold in the mirrors. Not sure if this is the best way to submit feature requests or not. Thanks for the terrific work on this project!

**Comment by tmack8001** [comment]
This request would require Rivian having a cloud command operation exposed for us to control. Today we are limited to what is supported and exposed through the mobile application. Unfortunately, we can't provide this control yet.

**Comment by AlanJohnWilliams** [comment]
Makes sense. Thanks. I'll send a request into Rivian for app feature request to control the mirrors. Any pointers on the best place to send such requests into Rivian? Through my guide?

**Comment by tmack8001** [comment]
For Rivian feedback there are 4 options... 
1. Add a "wish" to the community hosted owner feedback roadmap on, https://riviansoftware.canny.io/ (and up vote other owner submitted items)
2. Email your guide - mileage may vary here
3. Install Mobile version 2.5 and "share feedback" (scroll to the buttom of the experience)
4. Create a support ticket - some guides tell you to do this, then when you do and talk with support they say "this is how it was built/intended to be used, this isn't a support ticket" 🤷 

Personally I do num 1 and num 2 though may replace my continual messaging to my guide to be only for "bugs" vs new features and pass that off to num 3 which just released this week.
